src/staging/src/pages/alerts.js

```diff
@@ -292,13 +299,14 @@ const PublicAdvisoryPage = ({ data }) => {
   // If the filter changes, set data as old and get new data
   useEffect(() => {
     if (isNewFilter) {
-      let aType = checkAdvisoryType()
-      setAdvisoryType(aType)
-      setIsDataOld(true)
+      let aType = getAdvisoryTypeFromUrl();
+      updatePageTitle(aType);
+      setAdvisoryType(aType);
+      setIsDataOld(true);
 
       setPageIndex(1) // reset page back to 1
 
-      let q = getApiQuery(aType)
+      let q = getApiQuery(aType);
```

Comment:
We seem to be starting to mix code endings with and without `;`.  We should maintain code consistency throughout, instead of making the problem worse.

Reply:
done